src/test/java/TestOrderDaoImpl/OrderDaoImplTest.java

```diff
@@ -6,25 +6,37 @@
 
 import java.math.BigDecimal;
 import java.sql.SQLException;
+import java.util.Set;
 
 import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
 
 public class OrderDaoImplTest {
     private static final BigDecimal ALREADY_EXIST_ORDER = BigDecimal.valueOf(113042);
 private OrderDaoImpl orderDao = new OrderDaoImpl();
     @Test
     public void testFindOrderById() throws SQLException {
+        System.out.println("findOrderById ____________________");
         Order order = orderDao.findOrderById(ALREADY_EXIST_ORDER);
         System.out.println(order);
         assertNotNull(order);
+
     }
 
     @Test
-    public void testFindOrderByIdNull() throws SQLException {
-        Order order = orderDao.findOrderById(BigDecimal.valueOf(3));
-        System.out.println(order);
-        assertNotNull(order);
-    }
+    public void testGetAllOrders() throws SQLException {
+        System.out.println("testGetAllOrders ____________________");
+        Set<Order> orders = orderDao.getAllOrders ();
+        System.out.println(orders);
+        assertTrue(orders.size() > 25);
 
+    }
+    @Test
+    public void testGetAllOrdersInnerJoin() throws SQLException {
+        System.out.println("testGetAllOrdersInnerJoin ____________________");
+        Set<Order> orders = orderDao.getAllOrdersInnerJoin ();
+        System.out.println(orders);
+        assertTrue(orders.size() > 25);
+    }
```

Comment:
Also

Reply:
Good test in our case - cheak that results in both methods are same